examples/main.tf

```diff
@@ -2,10 +2,10 @@ provider "aws" {
   region = "eu-west-1"
 }
 
-module "aws_ecr" {
-#  source = "git@github.com:ministryofjustice/cloud-platform-terraform-ecr-credentials.git"
+module "example_team_ecr_credentials" {
+  #  source = "git@github.com:ministryofjustice/cloud-platform-terraform-ecr-credentials.git"
   source = "../"
```

Comment:
The example should use the [git repo as a remote source](https://www.terraform.io/docs/modules/sources.html#github), with a `ref` attribute pinning this to `master`, e.g:

`source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=master"`

Reply:
this will be change in the next pr as it affects the example since now there is no code in master.